Jenkinsfile

```diff
@@ -210,16 +210,16 @@ sed -e 's#{BUILD_BRANCH}#${env.BRANCH_NAME}#g' \
           """
              mkdir -p zowe-build/${env.BRANCH_NAME}_${env.BUILD_NUMBER} &&
              cd zowe-build/${env.BRANCH_NAME}_${env.BUILD_NUMBER} &&
-             git clone --branch master https://github.com/zowe/zowe-dockerfiles.git
-             cd zowe-dockerfiles/dockerfiles/zowe-release/s390x/zowe-v1-lts &&
+             git clone --branch ${env.BRANCH_NAME} https://github.com/zowe/zowe-install-packaging.git
```

Comment:
We can zip the containers folder and upload to zLinux server to avoid picking github url and branch name manually. This will always make sure the code on zLinux is always same as current build. If we checkout branch again, there is a chance the branch is updated during the build, then zLinux version has latest code, but the other docker has older version.

Also maybe we need to sanitize the branch name to make it safe to be a directory name, or create tmp folder with random alphanums.

Based on the process, making zLinux a Jenkins node could be better, so Jenkins can handle the folders, checkout etc. But anyway this way should work.